internal/app/machined/pkg/runtime/v1alpha1/v1alpha1_sequencer.go

```diff
@@ -131,14 +131,22 @@ func (*Sequencer) Install(r runtime.Runtime) []runtime.Phase {
 				"install",
 				Install,
 			).Append(
-				"mountBoot",
+				"mountSystem",
 				MountBootPartition,
```

Comment:
not for this PR, but can we/should we skip mounting `/boot`? it looks like we almost never need it now? (or do we still need to fetch meta?)

Reply:
:thinking: The only times that we need to now is when we perform an upgrade.